src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Action;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ActionHandler;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.impl.BalanceAction;
+import core.basesyntax.service.impl.PurchaseAction;
+import core.basesyntax.service.impl.ReturnAction;
+import core.basesyntax.service.impl.SupplyAction;
+import core.basesyntax.strategy.ActionStrategy;
+import core.basesyntax.strategy.ActionStrategyImpl;
+import core.basesyntax.strategy.DataConverter;
+import core.basesyntax.strategy.DataConverterImpl;
+import core.basesyntax.strategy.MyFileReader;
+import core.basesyntax.strategy.MyFileReaderImpl;
+import core.basesyntax.strategy.MyFileWriter;
+import core.basesyntax.strategy.MyFileWriterImpl;
+import core.basesyntax.strategy.StorageReportGenerate;
+import core.basesyntax.strategy.StorageReportGenerateImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        final String fromFile = "src/main/resources/report.csv";
+        final String toFile = "src/main/resources/new_report.csv";
```

Comment:
Avoid using hardcoded values for file paths. It's better to pass these as arguments to the `main` method or read them from a configuration file or environment variable.

Reply:
make them constants (+static, move to class level)